### ISSUE TITLE
welcome.py error

### ISSUE THREAD

**Issue by trankietmashiro** [SICKAG/sick_visionary_samples#1]
Hi, 
I am trying to run python3 -m visionary_welcome.python.welcome -i192.168.1.10 -d"Visionary-T Mini" to connect to my Visionary-T mini. I meet the error connection refused. How I should solve this problem? Thank you so much for your help!

**Comment by zimmefaSICKAG** [comment]
Hello,
based on the given information: Can you confirm that you used the correct IP address? Further, can try to also put the IP address in quotation marks?

It would help us if you can provide further information such as which OS you use and/or which shell? What is the exact error message? Are you sure that you are calling the above command from the correct folder such that the relative path, e.g. to the "base" folder is correct?

**Comment by trankietmashiro** [comment]
Thank you for your help! I have figured out my problem which related to IP setup. I opened SOPAS ET on windows and see 2 different IP addresses: the camera IP and PC network adapter IP. The problem was solved by changing the ethernet IP to be similar to PC network adapter IP then running with camera IP I saw on SOPAS ET.

Btw I use Ubuntu 20.04. So sorry I forget the exact error message since I solved it some weeks ago.
